lib/v2/navigation/navigation_service.dart

```diff
@@ -206,6 +213,7 @@ class NavigationService {
       return MaterialPageRoute(
         settings: settings,
         builder: (context) => appRoutes[routeName]!(arguments),
+        fullscreenDialog: fullScreenRoutes.contains(routeName),
```

Comment:
I knew that it was in the browser where the transitions are defined.

Reply:
Yeah thanks for that tip!